lib/Mojo/Promise.pm

```diff
@@ -56,7 +56,20 @@ sub catch { shift->then(undef, shift) }
 
 sub clone { $_[0]->new->ioloop($_[0]->ioloop) }
 
-sub finally { shift->_finally(1, @_) }
```

Comment:
The `_finally(1,` case it not used at all anymore, so you are leaving dead code behind.

Reply:
Not even sure if `_finally` is even needed anymore.